tests/endpoints/v1/test_features.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+
+from predicthq.endpoints.v1.features.schemas import FeatureResultSet
+from tests import with_mock_client, with_mock_responses, with_client
+
+
+class FeaturesTest(unittest.TestCase):
+
+    @with_mock_client()
+    def test_request_params_dicts(self, client):
```

Comment:
The endpoints tests are used as "kitchen sinks" for confirming that all params are properly passed to the client. Do you have all the params here?

Any reason for not adding the underscores version?

Reply:
Added more tests with all params and  underscores version